src/main/java/io/jenkins/plugins/folderauth/FolderAuthorizationStrategyManagementLink.java

```diff
@@ -430,4 +438,23 @@ public void doRemoveSidFromAgentRole(@QueryParameter(required = true) String rol
         FolderAuthorizationStrategyAPI.removeSidFromAgentRole(sid, roleName);
         redirect();
     }
+
+    @GET
+    @Restricted(NoExternalUse.class)
+    public void doAuthorizationStrategy(StaplerResponse response) throws IllegalStateException, IOException {
+        Jenkins jenkins = Jenkins.get();
+        jenkins.checkPermission(Jenkins.ADMINISTER);
+        AuthorizationStrategy strategy = jenkins.getAuthorizationStrategy();
+        if (!(strategy instanceof FolderBasedAuthorizationStrategy)) {
+            throw new IllegalStateException("Folder Based Authorization Strategy is not active.");
+        }
+        response.setContentType("application/json");
+        response.setCharacterEncoding("utf-8");
+        Writer writer = response.getWriter();
+        FolderBasedAuthorizationStrategyWrapper wrapper = new FolderBasedAuthorizationStrategyWrapper((FolderBasedAuthorizationStrategy) strategy);
+        JsonGenerator generator = jsonFactory.createGenerator(writer);
```

Comment:
`net.sf.json` was able to convert the `AuthorizationStrategy` to JSON but during the process it was generating over 1000 lines of warnings. I think this is a better solution. Please let me know if there is some better alternative.